gini-internal-payment-sdk/library/src/main/java/net/gini/android/internal/payment/paymentComponent/PaymentComponent.kt

```diff
@@ -0,0 +1,286 @@
+package net.gini.android.internal.payment.paymentComponent
+
+import android.content.Context
+import androidx.annotation.VisibleForTesting
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import net.gini.android.core.api.Resource
+import net.gini.android.health.api.models.PaymentProvider
+import net.gini.android.internal.payment.GiniInternalPaymentModule
+import net.gini.android.internal.payment.paymentProvider.PaymentProviderApp
+import net.gini.android.internal.payment.paymentProvider.getPaymentProviderApps
+import org.slf4j.LoggerFactory
+
+/**
+ * The [PaymentComponent] manages the data and state used by every [PaymentComponentView], the [MoreInformationFragment],
+ * and the [BankSelectionBottomSheet].
+ *
+ * It requires a [GiniMerchant] instance and a [Context] (application or activity) to be created.
+ */
+class PaymentComponent(@get:VisibleForTesting internal val context: Context, @get:VisibleForTesting internal val paymentModule: GiniInternalPaymentModule, private var configuration: PaymentComponentConfiguration = PaymentComponentConfiguration()) {
```

Comment:
[Clarification] Hi @llevente , do you think we could pass in an instance of `GiniHealthApi` and the language locale instead of the whole `GiniInternalPaymentModule` object?
Can the language be changed dynamically after initialising the `PaymentComponent` ? Or is it done when the app is started?